common-gradle-enterprise-gradle-configuration/settings.gradle

```
@@ -21,16 +21,23 @@ buildCache {
         enabled = true
     }
 
-    remote(HttpBuildCache) {
-        url = 'https://enterprise-samples.gradle.com/cache/' // adjust to your GE server, and note the trailing slash
+    // Use the Gradle Enterprise connector's access key based authentication
+    remote(gradleEnterprise.buildCache) {
+        enabled = true
+        push = isCI
+    }
+
+    // Use Gradle's built-in HTTP connector
+    /* remote(HttpBuildCache) {
```

I would put this on a separate line like so:
```suggestion
    /**
    remote(HttpBuildCache) {
```